apps/backend/app.py

```diff
@@ -139,14 +139,26 @@ def frange(start, stop, step=None):
         count += 1
 
 def gen_list(otherVar, paramspos):
+    print('before')
+    try:
+        step = otherVar['step']
+        if step == "":
+            if(otherVar['type'] == 'integer'):
+                step = 1
+            elif(otherVar['type'] == 'float'):
+                step = 0.1
+    except:
```

Comment:
Catch specific exception type? https://www.programiz.com/python-programming/exception-handling#:~:text=Catching%20Specific%20Exceptions%20in%20Python&text=We%20can%20specify%20which%20exceptions,in%20case%20an%20exception%20occurs.